src/mpi/coll/iallgather/iallgather_tsp_brucks.c

```diff
@@ -76,13 +79,14 @@ MPIR_TSP_Iallgather_sched_intra_brucks(const void *sendbuf, MPI_Aint sendcount,
 
     /* Step1: copy own data from sendbuf to top of recvbuf. */
     if (is_inplace && rank != 0)
-        MPIR_TSP_sched_localcopy((char *) recvbuf + rank * recvcount * recvtype_extent,
-                                 recvcount, recvtype, tmp_recvbuf, recvcount, recvtype, sched, 0,
-                                 NULL);
+        mpi_errno = MPIR_TSP_sched_localcopy((char *) recvbuf + rank * recvcount * recvtype_extent,
+                                             recvcount, recvtype, tmp_recvbuf, recvcount, recvtype,
+                                             sched, 0, NULL, &vtx_id);
```

Comment:
Can we add explicit `{...}` to the if/else branches? Without it it's difficult to read because of the multiline split and it is easy to make future mistakes.

Reply:
Oh, also, could you squash the commits since they are practically the same change albeit over multiple algorithms.

Reply:
These are old code, I can add braces here. But not sure if there are more of such statements.